packages/KTable/KTable.vue

```diff
@@ -35,7 +36,15 @@
               :name="value.key"
               :row="row"
               :rowKey="rowIndex"
-              :rowValue="row[value.key]">{{ row[value.key] }}</slot>
+              :rowValue="row[value.key]">
+              <a
```

Comment:
I think that you either need to wrap the entire row, which if so need to make sure you can still click buttons in the row without navigating. If that doesn't work I think we decided to leave as is and not use a native anchor tag

Reply:
why do we need to wrap the entire row? the row click is handled by `handleRowClick` which is implementation dependent but that way we can click the entire row as well as click any individual links inside of that row if we use this anchor tag here. Also wrapping the whole row with an `a` tag doesn't work well.

Reply:
let's chat with @darrenjennings. My understanding when we chatted was we were going to try and wrap the row if the prop was present _instead_ of the click handler to get native functionality. We could always just slot the anchor, theres no need to v-if one in the template. The thought was if a native link around the row doesn't work then we won't implement that. And we would document the user can add meta key support in their implementation

Reply:
whats the benefit of having the native click? this implementation works and fulfills all the requirements doesn't it? do we need to make it more complex? yeah in the example the meta key support is all handled in the implementation but that is going by the row click handler

Reply:
The idea of the anchor tag was to to get native right click/meta click support, and things like link url preview, etc. The discussion was _try_ to wrap in a link. If it doesn't work don't do it and just use the @row/@column handler

Reply:
yeah, I think the main problem with wrapping it is it completely messes up the table styling so I went with the row handler

Reply:
yeah I mentioned that might be a problem with wrapping the entire row. I wasn't sure what would happen but it might not be good. If its completely jacked with wrapping I'd say remove and just leave the updated click handler. Just add documentation to show how a user could hijack meta click